crypto/crypto.go

```diff
@@ -250,6 +259,30 @@ func Keccak256(v ...[]byte) []byte {
 	return h.Sum(nil)
 }
 
+// Keccak256Hash calculates and returns the Keccak256 hash of the input data,
+// converting it to an internal Hash data structure.
+func Keccak256Hash(v ...[]byte) (hash types.Hash) {
+	h := NewKeccakState()
+	for _, b := range v {
+		h.Write(b)
+	}
+
+	h.Read(hash[:])
+
+	return hash
+}
+
+// NewKeccakState creates a new KeccakState
+func NewKeccakState() KeccakState {
+	keccakState, isOk := sha3.NewLegacyKeccak256().(KeccakState)
+	if !isOk {
+		// this is only for linter, this will never happen
+		panic("newLegacyKeccak256 does not implement KeccakState interface")
```

Comment:
Weird to introduce a panic here specially if it's not going to happen, can we just use linting exclusion directive?

Reply:
Fixed [97f3d78](https://github.com/0xPolygon/polygon-edge/pull/796/commits/97f3d78fdf9f276162c65df6b07b1d855b190e3d)